Swift/replace.playground/Contents.swift

```diff
@@ -0,0 +1,17 @@
+import Foundation
+
+let helloString = "Hello, World!" // Defining initial string
+
+// Initializing regular expression
+var replaceRegExp: NSRegularExpression?
+
+do {
+    replaceRegExp = try NSRegularExpression(pattern: "World", options: .allowCommentsAndWhitespace)
```

Comment:
If are not stopping an execution on `NSRegularExpression` initialization failure, you may replace `replaceRegExp`  declaration with: `let replaceRegExp = try? NSRegularExpression(pattern: "World", options: .allowCommentsAndWhitespace)`.